pkg/cli/root_test.go

```diff
@@ -28,9 +28,8 @@ Usage: jvsctl COMMAND
   api           Perform API operations
   public-key    Perform public-key operations
   rotation      Perform rotation operations
-  token         Generate a justification token
+  token         Perform token operations
```

Comment:
Can you also update https://github.com/abcxyz/jvs/blob/main/prober/prober.sh to reflect the change?

Reply:
maybe https://github.com/abcxyz/jvs/blob/main/docs/cli.md#authentication as well?

Reply:
Updated https://github.com/abcxyz/jvs/blob/main/prober/prober.sh in a new commit. PTAL :D